TodoApi/appsettings.Development.json

```diff
@@ -1,9 +1,13 @@
 {
-  "Logging": {
-    "LogLevel": {
+  "Serilog": {
+    "ElasticSearchUrl": "http://localhost:9200",
+    "SeqUrl": "http://localhost:5341",
```

Comment:
Remove these by default. These should be specified in the env of the docker compose file.

```suggestion
```